Dockerfile

```diff
@@ -1,8 +1,59 @@
-ARG VERSION=unspecified
+# Official Docker images are in the form library/<app> while non-official
+# images are in the form <user>/<app>.
+FROM docker.io/library/python:3.13.1-alpine3.20 as compile-stage
```

Comment:
Are the warnings here addressed elsewhere?  If not then we may as well address them here.